tests/codeception/backend/acceptance/LoginCept.php

```diff
@@ -20,7 +20,6 @@
 $I->see('Password cannot be blank.', '.help-block');
 
 $I->amGoingTo('try to login with wrong credentials');
-$I->expectTo('see validations errors');
```

Comment:
Here `expectTo` should be kept for sure. It should be placed after `amGoinfTo` to compose scenario description.